src/components/shared/misc/SummarisedMultiSelector.vue

```diff
@@ -0,0 +1,220 @@
+
+        name: 'croud-summarised-multi-selector',
+
+        props: {
+            /**
+            * Pass an array of options to display in the dropdown
+            */
+            options: {
+                type: Array,
+                default() {
+                    return []
+                },
+            },
+
+            /**
+            * Maps the dropdown to the title/value fields on your options array.
+            */
+            fields: {
+                type: Object,
+                default() {
+                    return {
+                        title: 'name',
+                        value: 'id',
+                    }
+                },
+            },
+
+            /**
+            * The current value of the dropdown, v-model alias
+            */
+            value: {
+                type: [String, Array],
+                default() {
+                    return []
+                },
+            },
+
+            /**
+            * An optional placeholder for the dropdown
+            */
+            placeholder: {
+                type: String,
+                default() {
+                    return ''
+                },
+            },
+
+            /**
+            * Default text for the dropdown summary popup
+            */
+            defaultSummary: {
+                type: String,
+                default() {
+                    return 'Select an item'
+                },
+            },
+
+            /**
+            * Semantic ui dropdown configuration
+            */
+            dropdownSettings: {
+                type: Object,
+                default() {
+                    return {
+                        useLabels: false,
+                        showOnFocus: true,
+                        onHide: () => this.$emit('dropdown-hidden'),
+                    }
+                },
+            },
+
+            /**
+            * If the component should be in read only mode
+            */
+            readOnly: {
+                type: Boolean,
+                default() {
+                    return false
+                },
+            },
+
+            /**
+            * If the component should be inline
+            */
+            inline: {
+                type: Boolean,
+                default() {
+                    return true
+                },
+            },
+
+            /**
+            * If the component should be fluid
+            */
+            fluid: {
+                type: Boolean,
+                default() {
+                    return true
+                },
+            },
+        },
+
+        computed: {
+            model: {
+                get() {
+                    return this.value
+                },
+
+                set(val) {
+                    this.$emit('input', val)
+                },
+            },
+
+            summary() {
+                return this.model.length ? this.model.split(',').join(', ') : this.defaultSummary
```

Comment:
i have fixed the array issue, it will display whatever is set to be the value in the dropdown, maybe i can mention this in the readme, unless you have a better implementation suggestion?